libs/commons/Common.ml

```diff
@@ -1219,6 +1219,16 @@ let files_of_dir_or_files_no_vcs_nofilter xs =
          else [ x ])
   |> flatten
 
+let chdir_to_repo_root () =
+  let rec parent changed =
+    if Sys.getcwd () = "/" then invalid_arg "couldn't find semgrep root"
+    else if not (Sys.file_exists ".git" && Sys.is_directory ".git") then (
```

Comment:
If it's git related, move it to Git_wrapper.ml which actually already contains an is_git_repo() function.